### ISSUE TITLE
CTC is not updating self.TrackCTRLSignal._train_info in self.update()

### ISSUE THREAD

**Issue by ldt9** [itsmekhalidb/PlanesOverTrains#19]
Line 128 of CTC.py
update function

self.create_departures() does not add a new train ID with info to the dictionary, returns an empty dictionary upon departure

**Comment by ldt9** [comment]
@itsmekhalidb Can you take a look at this ASAP. We need to make sure your table that shows up in your UI is being placed in the dictionary because I wasn't seeing it when I printed out self.TrackCTRLSignal._train_info in your update function.

**Comment by itsmekhalidb** [comment]
I think this might be an issue with the track controller update function. I added a print statement in mine in the update function and it looked like it added the authority and train ids. I still need to put commanded speed in but it's passing.
![image](https://github.com/itsmekhalidb/PlanesOverTrains/assets/130712144/2f32e529-0e9a-46c6-8e85-a7c0380c708c)

When I tried adding the print statement to the track controller update, it showed the curly braces once then said Can't Update and stopped printing.
